k-distribution/src/main/scripts/bin/krun

```diff
@@ -521,10 +521,23 @@ if [ "$backendName" = "llvm" ]; then
   fi
 
   keepTempsIfDryRun "$tempDir" "$expanded_input_file" "$kore_output"
-  set +e
-  execute $cmdprefix "$kompiledDir/interpreter" "$expanded_input_file" "${depth:--1}" "$kore_output" $interpreter_flags
-  result=$?
-  set -e
+  if $search; then
+    if [ -n "$pattern" ]; then
+      error "Backend llvm does not support search patterns yet."
+    fi
+    if [[ $searchType != "FINAL" ]]; then
+      error "Backend llvm does not support --search-all, --search-one-or-more-steps, or --search-one-step yet"
+    fi
+    set +e
+    execute $cmdprefix "$kompiledDir/search" "$expanded_input_file" "${depth:--1}" "$kore_output" $interpreter_flags
+    result=$?
+    set -e
```

Comment:
```suggestion
    result=0
    execute $cmdprefix "$kompiledDir/search" "$expanded_input_file" "${depth:--1}" "$kore_output" $interpreter_flags \
      || result=$?
```

Or just set `result=0` at the top of the file, since it seems to only be used here.

Can you also change the remaining cases where this pattern is used? We sohuld just leave `-e` set, and use `|| result=$?` if we want to record a bad exit-code _and_ prevent failure.